src/Docker.DotNet/DockerClientConfiguration.cs

```diff
@@ -16,8 +16,14 @@ public class DockerClientConfiguration : IDisposable
 
         private static Uri LocalDockerUri()
         {
-            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-            return isWindows ? new Uri("npipe://./pipe/docker_engine") : new Uri("unix:/var/run/docker.sock");
+            var dockerHostVar = Environment.GetEnvironmentVariable("DOCKER_HOST");
```

Comment:
This is new support for an env in the command window right?

Reply:
I checked online the origin for this and haven't found a suitable answer. Reverting in next commit